osi_detectedobject.proto

```diff
@@ -93,16 +93,20 @@ message DetectedObject
     // The estimated probabilities for the object to belong to a specific class.
     //
     optional ClassProbability class_probability = 17;
-
-    // The estimated pedestrian head pose.
+    
+    // Pedestrian head pose for behaviour prediction. Describes the head
+    // orientation w.r.t. the host vehicle orientation.
+    // x-axis of head_pose is pedestrian's straight ahead viewing direction.
```

Comment:
Proposed extended definition for reduced ambiguity:
``The x-axis of the right-handed head frame is pointing along the pedestrian's straight ahead viewing direction and the z-axis is pointing upwards.``

Reply:
Is definition z-axis upwards neccessary? Or is it an unintended constraint!

Reply:
// The x-axis of the right-handed head frame is pointing along the 
    // pedestrian's straight ahead viewing direction and the z-axis is pointing 
    // upwards (cranial direction [1] i.e. to pedestrian's skull cap).
    //
    // \par References: 
    // [1] https://en.wikipedia.org/wiki/Anatomical_terms_of_location

Reply:
I think the minimum information required about the head attitude is yaw (pitch can probably be neglected and roll is irrelevant in my opinion). Now, since we define attitude by intrinsic rotations, the direction of the z-axis is important! Whether it points upwards or downwards basically determines wheter a positive yawing motion corresponds to a movement of the head towards the left or towards the right.
Good idea to mention the cranial direction!